R/readQFeatures.R

```diff
@@ -46,13 +52,11 @@
 ##' input (see next section).
 ##'
 ##'
-##' ## Multi-set case
+##' @section Multi-set case:
```

Comment:
Not sure we need a `@section` tag here, as we want this to be a sub-section within the Details section, not a new one.